definitions/virtual_machine.md

```diff
@@ -6,10 +6,11 @@ category: Technology
 ## Virtual Machine
 
 ### What it is
-A virtual machine enables a computer system to host other computer systems as guests on the same core system. The host system separates these secondary systems and limits the CPU usage, RAM, and filesystem size and access. With this separation, processes can be executed using different operating systems on the same physical machine.
+A virtual machine enables a computer system to host other computer systems as guests on the same core system. The host system separates these secondary systems and limits the CPU usage, RAM, and filesystem size and access. It will expose virtualised hardware resources (IRQ controllers, Timers, PCI devices) or will emulate them, depending on its capabilities and configuration.
```

Comment:
At a minimum omit the part about virtualized hardware. I'd also like to see the overall language made a lot simpler. 
Ex: What is a virtual machine? It's a computer that doesn't share a 1 to 1 relationship with a piece of hardware. A virtual machine relies on [virtualization](link) to run on a physical server alongside other virtual machines.